docs/nftfair/introduction.md

```diff
@@ -0,0 +1,59 @@
+# Introduction
+
+Automata NFTFair is A NFT launch platform designed around principles of fairness and efficacy for a community-led experience is how we would describe NFTFair. Metadata exploits pre-sniping are all predatory practices that make it unfair for the average user to get their hands on a new collection, much less obtain rare or lucrative items.
```

Comment:
seems there are grammar issues?

Reply:
actually, that's not my work, I copy it from sam's medium, hh